src/utils/checkParams.ts

```diff
@@ -1,11 +1,13 @@
 /**
```

Comment:
After packing this file changes to be an empty file. Can we delete it after packing?

Reply:
Yes, I can.

But I think I'll do it by adding a file deletion pipe in gulp, is that okay?

Reply:
> Yes, I can.
> 
> But I think I'll do it by adding a file deletion pipe in gulp, is that okay?

If this will not make an error it could be done.

I see some files imported the function although it is empty.

Reply:
> > Yes, I can.
> > But I think I'll do it by adding a file deletion pipe in gulp, is that okay?
> 
> If this will not make an error it could be done.
> 
> I see some files imported the function although it is empty.

Maybe I can fix it all. I'll try it now

Reply:
@ZLY201 This problem is solved using eslint and babel I tested it and it works well !